### PR TITLE
Image uploads

### PR DESCRIPTION
**Pull request by GaretJax**

### CODE REVIEW

File: aldryn_forms/cms_plugins.py [hunk 1]
```diff
@@ -494,6 +496,39 @@ def form_pre_save(self, instance, form, request, **kwargs):
...
+class ImageField(FileField):
+    name = _('Image upload field')
+
+    model = models.ImageUploadFieldPlugin
+
+    form = ImageFieldForm
+    form_field = RestrictedImageField
+    form_field_widget = RestrictedImageField.widget
+    fieldset_general_fields = Field.fieldset_general_fields + [
+        'upload_to',
```
Comment (stefanfoulis) [inline aldryn_forms/cms_plugins.py]: This might be a security issue. Is this exposed to the user?

Comment (stefanfoulis) [inline aldryn_forms/cms_plugins.py]: ah, nvm. I didn't see that this is a ForeignKey to FilerFolder. So probably not a security problem.

Comment (GaretJax) [inline aldryn_forms/cms_plugins.py]: It's the folder where the files fill be uploaded to. I take care to set `is_public` to false for those files. Is that enough?

**Comment by czpython** [comment]
LGTM

File: aldryn_forms/cms_plugins.py (hunk 1 above)
Comment (stefanfoulis) [inline aldryn_forms/cms_plugins.py]: Files have a random component in the url anyway, so it is already virtually impossible to access the files without the full url.

For the files to be password protected, the project has to configure django-filer to use a secure storage backend and serve the files through django (http://django-filer.readthedocs.org/en/latest/secure_downloads.html). Also permissions have to be set on the folder.

**Pull request merged**